distributed/deploy/ssh.py

```diff
@@ -95,13 +95,13 @@ async def start(self):
         result = await self.connection.run("uname")
         if result.exit_status == 0:
             set_env = 'env DASK_INTERNAL_INHERIT_CONFIG="{}"'.format(
-                serialize_for_cli(dask.config.global_config)
+                serialize(dask.config.global_config)
```

Comment:
Nitpick: We might consider using the full `dask.config.serialize` here instead of just `serialize` in order to not get confused with `distributed.protocol.serialize`. At least for me personally, when I see `serialize` in `distributed` code I immediately think about `distributed.protocol.serialize`